File: app/src/main/java/de/bitb/astroskop/ui/main/profiles/IProfilesView.java
package de.bitb.astroskop.ui.main.profiles;

import de.bitb.astroskop.model.Profile;
import de.bitb.astroskop.ui.base.IBaseView;

interface IProfilesView extends IBaseView{
    void openProfile(Profile profile);

    void openDeleteDialog(Profile profile);

    void refreshView();
}

File: app/src/main/java/de/bitb/astroskop/ui/main/profiles/IProfilesView.kt
package de.bitb.astroskop.ui.main.profiles

import de.bitb.astroskop.model.Profile
import de.bitb.astroskop.ui.base.IBaseView

internal interface IProfilesView : IBaseView {
    fun openProfile(profile: Profile)

    fun openDeleteDialog(profile: Profile)

    fun refreshView()
}
